src/docs/quicktips/eliminate-js.md

```diff
@@ -16,7 +16,7 @@ Read more at the [GitHub API documentation](https://developer.github.com/v3/repo
 
 This is a bit different from our client-side implementation because this data is only updated as often as your build runs. This is implemented using a global [JavaScript data file](/docs/data-js/) at `_data/github.js`.
 
-* Install new dependencies: `npm install node-fetch --save-dev`
+* Install new dependencies: `npm install node-fetch@2.0 --save-dev`
```

Comment:
Q: Should this be strictly "2.0.x", or should we relax it to `@2` (without pinning to a minor version)?

From my quick poking, it looks like there are several newer 2.x builds (w/ 2.6.7 from 2022-01-16, whereas "2.0" would most likely resolve to 2.0.0 from 2018-02-03):

```sh
npm info node-fetch time --json | grep "\"2\." | grep -v "\-alpha"

  "2.0.0": "2018-02-03T20:34:24.774Z",
  "2.1.0": "2018-03-05T04:41:12.676Z",
  "2.1.1": "2018-03-05T04:44:09.059Z",
  "2.1.2": "2018-03-25T20:55:00.498Z",
  "2.2.0": "2018-07-22T21:32:05.747Z",
  "2.2.1": "2018-11-05T09:44:41.523Z",
  "2.3.0": "2018-11-13T06:41:56.187Z",
  "2.4.0": "2019-04-26T17:24:20.869Z",
  "2.4.1": "2019-04-27T07:07:42.071Z",
  "2.5.0": "2019-05-01T05:08:07.621Z",
  "2.6.0": "2019-05-16T06:39:41.004Z",
  "2.6.1": "2020-09-05T13:00:44.110Z",
  "2.6.2": "2021-09-06T13:05:31.143Z",
  "2.6.3": "2021-09-20T21:52:54.550Z",
  "2.6.4": "2021-09-21T12:38:53.828Z",
  "2.6.5": "2021-09-22T12:53:43.259Z",
  "2.6.6": "2021-10-31T15:49:13.781Z",
  "2.6.7": "2022-01-16T12:18:38.923Z",
```

Reply:
Oooh, that is very cool. Also TIL;  `dist-tags` thanks Peter!